pkg/loop/internal/pb/ccip/pricegetter.proto

```diff
@@ -11,10 +11,23 @@ import "google/protobuf/empty.proto";
 // PriceGetter is a service that returns the price of a token in USD. It is a gRPC service adapter for the interface
 // [github.com/smartcontractkit/chainlink-common/chainlink-common/pkg/types/ccip/PriceGetter]
 service PriceGetter {
+	rpc IsTokenConfigured(TokenConfiguredRequest) returns (TokenConfiguredResponse) {}
```

Comment:
weird intention.

Reply:
Weird, Github diff being formatted like that

Actual code is as expected
![image](https://github.com/smartcontractkit/chainlink-common/assets/28818476/2890781c-f0ac-4e36-b170-100356527653)

Reply:
Oh, it's a tab vs. spaces from my code editor formatter. Fixed this